src/com/sun/jna/win32/W32StringUtil.java

```diff
@@ -0,0 +1,390 @@
+/* Copyright (c) 2020 Torbjörn Svensson, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.win32;
```

Comment:
Move this to internal - this whole changeset already is complex enough, I don't want to see this exposed. Oh and then it needs to be moved to platform.

Reply:
I tried that first, but since there was one class that is win32, but it's located outside platform, I couldn't do it.

Reply:
? - I don't understand what you mean here.

Reply:
Hmm. I tried having it in platform first, but I got some error. I just tried to move it back to platform.win32 and I do not see any errors any more, so I guess whatever I had in my workspace at that time is now resolved.
I'll move it if we still want to go this route.